tox.ini

```diff
@@ -9,22 +9,22 @@ commands =
     pytest
 
 [testenv:black]
-basepython = python3.8
+basepython = python3.9
```

Comment:
@jdavies-st recently pointed out to me setting`basepython` here has a downside, which is that we can't do this:

```
tox -e py38-black
```

We've been removing `basepython` in other repos, can you do that here too?